elections.md

```diff
@@ -1,55 +1,40 @@
-election activities while they serve as continuity members. Members of the
-bootstrap committee will refrain from endorsing or otherwise advocating for any
-candidate (with the exception that the members of the bootstrap committee may
-vote in the elections, and may choose to run in the 2019 election).
-
-### Members of Standing
+### Eligibility for voting
 
-Initial standing in the Kubernetes community will be defined by the union of:
-* SIG leads
-* Approvers and reviewers in any Kubernetes owned repositories
-* Anyone with write access to a Kubernetes owned repository
+Eligibility to vote for steering committee members is defined by:
 
-The bootstrap committee *explicitly* believes that this heuristic will be
-inaccurate and not represent the entire community. An exception form will be
-provided for people who have contributed to the project but may not meet these
-criteria. Exception applications will be approved by the bootstrap committee
-will vote on this eligibility, with simple majority determining eligibility.
-The exception process will be used as data points for refining the criteria for
-the future.
+* People who have at least 50 [contributions to the Kubernetes project over the past year as defined by devstats]
+  as of 2018-08-15. Contributions include GitHub events like creating issues,
+  creating pr's, reviewing PR's, commenting on issues, etc. For full details
+  see [the SQL query used by devstats for developer stats]
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. Acceptance will be defined by a simple majority
+  vote, and the criteria used during this process will be used to help refine
+  further elections.
 
-Within its first year, the Steering Committee will redefine these criteria
-around a more robust idea of community membership.
+It is the responsibility of the steering committee to refine these criteria prior
+to each election.
 
 ### Eligibility for candidacy
 
-Anyone may nominate either themselves or someone else to be a candidate in the
-election. To be ratified as a candidate, the nominee must accept the nomination
-and three Members of Standing (including the nominator, if she/he has standing)
-from three different employers, must endorse the nomination.  
+Eligibility for candidacy is defined by:
 
-Nominators are free to nominate as many people as they wish to. Members of
-Standing may endorse multiple nominees, but we expect endorsements to be in
-good faith.  If this turns out to be a problem, this will be reconsidered.
-
-### Eligibility for voting
+* Acceptance of a nomination, or self-nomination (anyone may nominate, anyone
+  may be nominated)
+* Sponsorship by three eligible voters from three different employers (the
+  candidate can self-sponsor if they are eligible to vote)
 
-All Members of Standing are eligible to vote for the steering committee
-members. 
+Nominators are free to nominate as many people as they wish to. Eligible
```

Comment:
yah, I'm confused as well ..

Reply:
Ack.  Let's s/sponsor/endorse, since that appears to be the term used last year

The way I read things:
- anyone can nominate
- anyone can be nominated
- only those eligible to vote can endorse

Reply:
That looks right @spiffxp